.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,115 @@
+id: 
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: 
+resource: repository
+disabled: false
+where: 
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description: Auto close "Area-NuGet" issues and add a comment pointing to NuGet repo
+      frequencies:
+      - daily:
+          time: 16:0
+      filters:
+      - isIssue
+      - isOpen
+      - hasLabel:
+          label: Area-NuGet
+      actions:
+      - addReply:
+          reply: >-
+            Thanks for creating this issue! We believe this issue is related to NuGet tooling, which is maintained by the NuGet team. Thus, we closed this one and encourage you to raise this issue in the [NuGet repository](https://github.com/NuGet/Home) instead. Don’t forget to check out [NuGet’s contributing guide](https://github.com/NuGet/Home/blob/dev/CONTRIBUTING.md#before-submitting-an-issue) before submitting an issue!
+
+
+            If you believe this issue was closed out of error, please comment to let us know.
+
+
+            Happy Coding!
+      - closeIssue
+    eventResponderTasks:
+    - if:
+      - payloadType: Pull_Request
+      - hasLabel:
+          label: Auto-Merge If Tests Pass
+      then:
+      - enableAutoMerge:
+          mergeMethod: Squash
+      description: Automatically merge pull requests
+    - if:
+      - payloadType: Pull_Request
+      - labelRemoved:
+          label: Auto-Merge If Tests Pass
+      then:
+      - disableAutoMerge

```

Comment:
when would this happen? Is this a case the label gets added when checks haven't finished, an issue is found in the PR, and so the label gets removed while the issue is getting resolved? Is the AutoMerge state persistent without this?

Reply:
Covered in my comment above. https://github.com/dotnet/sdk/pull/38507#discussion_r1483742695